### ISSUE TITLE
Additional PHPStan extensions by default

### ISSUE THREAD

**Issue by dkarlovi** [jakzal/phpqa#70]
You've mentioned (somewhere) in a PR / issue you're planning on shipping some PHPStan extensions by default.

Here are some worth while IMO:
- [Deprecation rules](https://github.com/phpstan/phpstan-deprecation-rules) (official)
- [Exception rules](https://github.com/pepakriz/phpstan-exception-rules) (community)
- [Strict rules](https://github.com/phpstan/phpstan-strict-rules)  (official)
- [Doctrine support](https://github.com/phpstan/phpstan-doctrine)  (official)
- [PHPUnit support](https://github.com/phpstan/phpstan-phpunit)  (official)
- [Symfony support](https://github.com/phpstan/phpstan-symfony)  (official)

**Comment by jakzal** [comment]
Since the extensions are disabled by default (correct me if I'm wrong), I don't see anything wrong with shipping at least the official ones.

**Comment by dkarlovi** [comment]
I agree. There are more, probably can include those too since they're mostly tiny:

https://github.com/phpstan/phpstan-nette
https://github.com/phpstan/phpstan-dibi
https://github.com/phpstan/phpstan-beberlei-assert
https://github.com/phpstan/phpstan-webmozart-assert

**Comment by jakzal** [comment]
@dkarlovi are you up for preparing a PR?

**Comment by dkarlovi** [comment]
@jakzal since the paths will be directly used by end users, what do you think about symlinking the extensions somehow?

For example, `/root/.composer/vendor-bin -> /extensions`?
